lib/private/Files/Storage/Wrapper/Encryption.php

```diff
@@ -598,6 +599,33 @@ protected function fixUnencryptedSize($path, $size, $unencryptedSize) {
 		return $newUnencryptedSize;
 	}
 
+	/**
+	 * fread_block
+	 *
+	 * This function is a wrapper around the fread function.  It is based on the
+	 * stream_read_block function from lib/private/Files/Streams/Encryption.php
+	 * It calls stream read until the requested $blockSize was received or no remaining data is present.
+	 * This is required as stream_read only returns smaller chunks of data when the stream fetches from a
+	 * remote storage over the internet and it does not care about the given $blockSize.
+	 *
+	 * @param handle the stream to read from
+	 * @param int $blockSize Length of requested data block in bytes
+	 * @return string Data fetched from stream.
+	 */
+	private function fread_block($handle, $blockSize): string {
```

Comment:
```suggestion
	private function fread_block($handle, int $blockSize): string {
```
add type hint for blockSize?

Reply:
@alanmeeson, can you address those? :)

Reply:
Type hint and cs:fix comments addressed.